src/arrayMethodUnshift.js

```diff
@@ -5,8 +5,17 @@
  *
  */
 function applyCustomUnshift() {
-  [].__proto__.unshift2 = function() {
-    // write code here
+  [].__proto__.unshift2 = function(...theArgs) {
+    let l = this.length;
```

Comment:
l => arrLen, or  length. 

You seem to be using this variable only in one place, so maybe you don't need a varabile at all?